tests/roles/firewalld

```diff
@@ -0,0 +1 @@
+../../
```

Comment:
Is this file necessary when we're running molecule?

Reply:
No, I added that when I was running Vagrant.  Should we include this in the event that someone wants to test with Vagrant, or is that redundant with molecule?